### PR TITLE
Restore trigger recommendations rn

### PR DESCRIPTION
**Pull request by AradCarmi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Review by dorschw — COMMENTED** [review]
Please add a unit test

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -234,6 +234,7 @@ class FileType(str, Enum):
     FileType.XDRC_TEMPLATE.value: XDRC_TEMPLATE_DIR,
     FileType.CORRELATION_RULE.value: CORRELATION_RULES_DIR,
     FileType.XSIAM_DASHBOARD.value: XSIAM_DASHBOARDS_DIR,
+    TRIGGER_RECOMMENDATION: TRIGGER_DIR,
```
Comment (dantavori) [inline demisto_sdk/commands/common/constants.py:237]:
```suggestion
    FileType.TRIGGER: TRIGGER_DIR,
```

File: demisto_sdk/commands/common/constants.py [hunk 2]
```diff
@@ -87,9 +87,9 @@
 XSIAM_DASHBOARD = "xsiamdashboard"
 XSIAM_REPORT = "xsiamreport"
 TRIGGER = "trigger"
+TRIGGER_RECOMMENDATION = "triggersrecommendation"
```
Comment (dantavori) [inline demisto_sdk/commands/common/constants.py:90]:
```suggestion
```

File: demisto_sdk/commands/update_release_notes/update_rn.py [hunk 3]
```diff
@@ -255,8 +254,6 @@ def execute_update(self) -> bool:
         changed_files = {}
         for packfile in self.modified_files_in_pack:
             file_name, file_type = self.get_changed_file_name_and_type(packfile)
-            if file_type in SKIP_RELEASE_NOTES_FOR_TYPES:
```
Comment (dantavori) [inline demisto_sdk/commands/update_release_notes/update_rn.py:258]: why deleted?

File: demisto_sdk/commands/common/constants.py [hunk 4]
```diff
@@ -87,9 +87,9 @@
...
 WIZARD = "wizard"
 XDRC_TEMPLATE = "xdrctemplate"
-
```
Comment (dantavori) [inline demisto_sdk/commands/common/constants.py:92]: please undo (MARKETPLACE_KEY_PACK_METADATA is not an entity)

**Review by dantavori — CHANGES_REQUESTED** [review]
after fixing, please make some tests to ensure the command is working as expected

File: demisto_sdk/commands/update_release_notes/update_rn.py (hunk 3 above)
Comment (AradCarmi) [reply demisto_sdk/commands/update_release_notes/update_rn.py:258]: The skip is implemented in hight functions.

File: demisto_sdk/commands/common/hook_validations/release_notes.py [hunk 5]
```diff
@@ -186,10 +187,12 @@ def validate_content_item_header(
             True if the content item is valid, False otherwise.
         """
         is_valid = True
-        entity_type = content_type.replace(" ", "")
-        entity_type = entity_type[:-1] if entity_type.endswith("s") else entity_type
+        entity_type = FILE_TYPE_BY_RN_HEADER.get(content_type, "")
 
-        content_type_dir_name = ENTITY_TYPE_TO_DIR.get(entity_type.lower(), entity_type)
+        # entity_type = content_type.replace(" ", "")
+        # entity_type = entity_type[:-1] if entity_type.endswith("s") else entity_type
```
Comment (dantavori) [inline demisto_sdk/commands/common/hook_validations/release_notes.py:193]:
```suggestion
```

**Review by dantavori — APPROVED** [review]

**Pull request merged**